<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>orgEdit</class>
 <widget class="QWidget" name="orgEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑组织</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>30</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>组织</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QListView" name="orgListView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>191</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="memberListView">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>191</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>当前成员</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="addMemberBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delMemberBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
